kuma/javascript/src/newsletter.jsx

```diff
@@ -167,13 +162,32 @@ export default function Newsletter() {
         }
     }, []);
 
+    if (!showNewsletter) {
+        return null;
+    }
+
+    if (showSuccessfulSubscription) {
+        return (
+            <section className="newsletter-container">
+                <div id="newsletter-thanks" className="newsletter-thanks">
+                    <h2>
+                        {gettext(
+                            'Thanks! Please check your inbox to confirm your subscription.'
+                        )}
+                    </h2>
+                    <p>
+                        {gettext(
+                            'If you haven’t previously confirmed a subscription to a Mozilla - related newsletter you may have to do so. Please check your inbox or your spam filter for an email from us.'
+                        )}
+                    </p>
+                </div>
+            </section>
+        );
+    }
+
     return (
         <section className="newsletter-container">
-            <div
-                id="newsletter-form-container"
-                className={showNewsletter ? 'newsletter' : 'hidden'}
-                aria-hidden={showNewsletter ? false : true}
-            >
+            <div id="newsletter-form-container" className="newsletter">
```

Comment:
Why bother with an ID?

Reply:
True, didn't even question those. I threw all the IDs that didn't serve a purpose out.